### ISSUE TITLE
Picker is closing on value change in iOS

### ISSUE THREAD

**Issue by varun-naharia** [markuswind/react-native-select-input-ios#70]
```
<SelectInput
                style={styles.picker}
                value={ounitr}
                options={unit}
/>
```

I am using above code for picker where unit is defined 

```
constructor(props) {
    super(props);
    this.state = {
      unit: [
        {label: 'mm', value: 'mm'},
        {label: 'cm', value: 'cm'},
        {label: 'meters', value: 'm'},
        {label: 'km', value: 'km'},
        {label: 'inches', value: 'in'},
        {label: 'feet', value: 'ft'},
        {label: 'yards', value: 'yd'},
        {label: 'miles', value: 'mi'},
        {label: 'nautical miles', value: 'nmi'},
      ],
      scaler: 1,
      scales: 12,
      oscaler: 1,
      oscales: 12,
      olengthr: 120,
      olengths: 10,
      ounitr: 'mm',
      ounits: 'mm',
      formula: '',
    };
    this.setOunitrValue = this.setOunitrValue.bind(this);
    this.setOunitsValue = this.setOunitsValue.bind(this);
  }
```

![vyBx8olr8s](https://user-images.githubusercontent.com/10987732/79483850-7355a680-8030-11ea-9675-419b6c86da70.gif)

**Comment by JoungSik** [comment]
The same thing is happening to me.

**Comment by varun-naharia** [comment]
I used different picker as I am unable find any solution for this problem

**Comment by markuswind** [comment]
I'll take a look at why this is happening.

**Comment by Kasendwa** [comment]
Hi @markuswind, I just spent a while trying to figure out where the issue was. I've made a PR (#71)  . Please review it and merge if all is well. It fixes the issue for me.

**Comment by markuswind** [comment]
Hi @Kasendwa, thanks for fixing this issue! Merged it, and released it in v2.0.5.
